handbook/engineering/distribution/goals.md

```diff
@@ -4,7 +4,7 @@ Goals are continuously updated and reviewed. If you find these goals do not refl
 
 ## Medium-term goals (~3-6 Months)
 
-### [Improve deployment pipeline UX](https://github.com/orgs/sourcegraph/projects/96)
+### [Improve internal deployment pipeline UX](https://github.com/orgs/sourcegraph/projects/96)
```

Comment:
Just a minor clarification, because I thought this was talking about the improved deployment process for customers at first (Dhall).